schemas/json/auth-v1.json

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$metadata": {
+    "uris": {
+      "jsonLdContext": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/auth-v2.json-ld",
```

Comment:
`jsonLdContext` and `jsonSchema` has different versions

Reply:
fixed

Reply:
that's ok